main.js

```diff
@@ -1,163 +1,384 @@
+        options: {
+            wifi: true,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: true,
+        },
+        originalStock: 6,
+        sold: 3,
     },
-    originalStock: 6,
-    sold: 3,
-  },
-  {
-    type: 'B2450HD',
-    name: 'LED TV',
-    brand: 'Brandt',
-    price: 109,
-    availableSizes: [24],
-    refreshRate: 60,
-    screenType: 'LED',
-    screenQuality: 'Full HD',
-    smartTv: false,
-    options: {
-      wifi: false,
-      speech: false,
-      hdr: false,
-      bluetooth: false,
-      ambiLight: false,
+    {
+        type: 'B2450HD',
+        name: 'LED TV',
+        brand: 'Brandt',
+        price: 109,
+        availableSizes: [24],
+        refreshRate: 60,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: false,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-  {
-    type: '32WL1A63DG',
-    name: 'HD TV',
-    brand: 'Toshiba',
-    price: 161,
-    availableSizes: [32],
-    refreshRate: 50,
-    screenType: 'LED',
-    screenQuality: 'Full HD',
-    smartTv: false,
-    options: {
-      wifi: false,
-      speech: false,
-      hdr: true,
-      bluetooth: false,
-      ambiLight: false,
+    {
+        type: '32WL1A63DG',
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-];
\ No newline at end of file
+];
+
+// Create a function to calculate how many tv's we have to sell
+const toSell = function (inventory) {
+    // Create a temporary original stock count
+    let originalStockCount = 0
+    // Create a temporary sold count
+    let soldCount = 0
+    // Looping over all the inventory inputs
+    for (let i = 0; i < inventory.length; i++) {
+        // Creating a temporary tv variable
+        let tmpInventory = inventory[i].originalStock
+        // Adding the original stock to the temporary stock variable
+        originalStockCount += tmpInventory
+    }
+    // Looping over all the inventory inputs
+    for (let i = 0; i < inventory.length; i++) {
+        // Creating a temporary sold variable
+        let tmpInventory = inventory[i].sold
+        // Adding the sold to the temporary stock variable
+        soldCount += tmpInventory
+    }
+    // Returning the original stock counts minus the sold count
+    return originalStockCount - soldCount
+}
+
```

Comment:
nice, je kan hem nog korter maken door de twee samen te voegen tot eén:
for (let i = 0; i < tvs.length; i++) {
    const unsold = tvs[i].originalStock - tvs[i].sold;
    total = total + unsold;
  }
  return total;